### ISSUE TITLE
Skin turgor low

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9917]
**Preferred term label:**
Skin turgor low

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Hypernatermia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000303
- label: Decreased skin turgor
- parents: [Abnormal skin morphology (HP:0011121)](https://hpo.jax.org/app/browse/term/HP:0011121)
- pmid: PMID:30705869
- def: Reduced skin turgor means reduced elastic recoil of the skin to its normal contour after being pinched in a fold.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9917
- synonyms:
